packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -234,11 +236,11 @@ module.exports = ctx => ({
     },
 
     company_logos: [
-      { name: 'Major League Baseball', icon: '/img/logos/baseball.png' },
-      { name: 'Splunk', icon: '/img/logos/splunk.png' },
-      { name: 'Adobe', icon: '/img/logos/adobe.png' },
-      { name: 'JetBlue', icon: '/img/logos/jetblue.png' },
-      { name: 'Experian', icon: '/img/logos/experian.png' },
+      { name: 'Major League Baseball', icon: '/img/logos/baseball.svg', blue: '/img/logos/blue/baseball.svg' },
+      { name: 'Splunk', icon: '/img/logos/splunk.svg', blue: '/img/logos/blue/splunk.svg' },
+      { name: 'Adobe', icon: '/img/logos/adobe.svg', blue: '/img/logos/blue/adobe.svg' },
+      { name: 'JetBlue', icon: '/img/logos/jetblue.svg', blue: '/img/logos/blue/jetblue.svg' },
+      { name: 'Experian', icon: '/img/logos/experian.svg', blue: '/img/logos/blue/experian.svg' },
```

Comment:
It feels like these icons only differ by their color, can we adjust the SVGs with `currentColor` and style the difference instead?

Reply:
These icons are inserted as src attribute in tag img, and can't be styled via css. If you can advise me on how to import them into vue component so I can insert them as svg I would be grateful. I couldn't manage how to do it myself.
And their source is too big to store them in config file as <svg>.

Reply:
Good point, and out of curiosity, is the "icon" version still in use anywhere with this PR?

Reply:
Yes, there are both black and blue versions of icons in design)